src/app/item-page/media-viewer/media-viewer-video/caption-info.ts

```diff
@@ -0,0 +1,4 @@
+export class CaptionInfo {
```

Comment:
Could we add some TypeDocs / comments to better describe this class?  It looks like its purpose is to store information about the captioning file, so that it can be placed in the HTML5 `<track>` tag.  So, we should add a brief note to describe how this is used.